README.md

```diff
@@ -1 +1,99 @@
-# heroku-scaler
+# Heroku Scaler
+
+The Heroku Scaler gem allows for free scaling of Heroku web dynos.
+It exposes 3 simple rake tasks that can be called from the free [Heroku
+Scheduler](https://elements.heroku.com/addons/scheduler).
+
+This is perfect for applications that see heavy use during known times of the day or week.  For example, we have some customers with business systems that see 99.9% of their traffic during regular business hours on the weekdays.  Since Heroku charges for dynos based on the number of seconds the dynos are running, scaling down application dynos during off-peak times can mean big savings.  Running your larger dyno setup during the 25% of normal business hours can save a bundle!  There are other tools that purport to do this but they often charge money and have, in our experience, been poorly supported.
```

Comment:
Whoa.  Great catch.  That was a horrible sentence.  Thanks for the recommendation!